File: core/src/main/java/in/koreatech/koin/core/networks/responses/BusResponse.java
package in.koreatech.koin.core.networks.responses;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import in.koreatech.koin.core.networks.entity.CityBus;

/**
 * Created by hyerim on 2018. 8. 13....
 */
public class BusResponse {
    @SerializedName("bus_number")
    @Expose
    public int busNumber;

    @SerializedName("remain_time")
    @Expose
    public int remainTime;

    @SerializedName("next_bus_number")
    @Expose
    public int nextBusNumber;

    @SerializedName("next_remain_time")
    @Expose
    public int nextRemainTime;


}

File: core/data/src/main/java/in/koreatech/core/networking/response/bus/BusResponse.kt
package `in`.koreatech.core.networking.response.bus

import com.google.gson.annotations.SerializedName
import `in`.koreatech.core.networking.response.bus.BusRemainTimeResponse

data class BusResponse(
    @SerializedName("bus_type") val busType: String,
    @SerializedName("now_bus") val nowBus: BusRemainTimeResponse? = null,
    @SerializedName("next_bus") val nextBus: BusRemainTimeResponse? = null
)